### DISCUSSION TITLE
Mesh mapping issue between BeamDyn and AeroDyn

### DISCUSSION THREAD

**Discussion by GBA217** [OpenFAST/openfast#2168] [Q&A]
I'm trying to run ElastoDyn + BeamDyn for IEA-15-240-RWT-Monopile and I encounter this error:

 FAST_InitializeAll:ED_Init:ED_SetParameters:SetPrimaryParameters: AllBldNdOuts option not
available in ElastoDyn when BeamDyn is used.  Turning off these outputs.
FAST_InitializeAll:InflowWind_Init:IfW_UniformWind_Init: Could not read upflow column in uniform
wind files. Assuming upflow is 0.
FAST_InitializeAll:Error allocating 895235120 bytes of memory for the AllOutData array.
FAST_InitializeAll:InitModuleMappings:BD_2_AD_BladeMotion(1):MeshMapCreate:CreateMotionMap_L2_to_L
2:CreateMapping_ProjectToLine2:Found close value for node 1. (9.22863E-05 m)
InitModuleMappings:AD_2_BD_BladeLoad(1):MeshMapCreate:CreateLoadMap_L2_to_L2:CreateMapping_Project
ToLine2:Found close value for node 1. (9.22863E-05 m)
InitModuleMappings:BD_2_AD_BladeMotion(2):MeshMapCreate:CreateMotionMap_L2_to_L2:CreateMapping_Pro
jectToLine2:Found close value for node 1. (1.00418E-04 m)
InitModuleMappings:AD_2_BD_BladeLoad(2):MeshMapCreate:CreateLoadMap_L2_to_L2:CreateMapping_Project
ToLine2:Found close value for node 1. (1.00418E-04 m)
InitMod

 OpenFAST encountered an error during module initialization.
 Simulation error level: FATAL ERROR

 Aborting OpenFAST.
 
 I understand this is an issue with mesh mapping between BeanDyn and AeroDyn. How do I fix it?

**Comment by jjonkman** [comment]
Dear @gorby217,

Actually, the mesh-mapping messages are simply warnings.  These warnings were similarly reported in the following forum topic: https://forums.nrel.gov/t/running-iea-15-240-rwt-with-beamdyn-instead-of-elastodyn/4153, where the actual error was the use of too large of a time step).

The error causing your simulation to abort is the error about allocating memory for the `AllOutData` array.  A similar question was asked and answered here: https://github.com/OpenFAST/openfast/issues/1922.

Best regards,
